src/components/FadeOutText/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { ReactNode, useEffect, useState } from 'react'
+
+interface Props {
+  children?: ReactNode
+}
+
+export default function FadeOutText({ children, ...props }: Props) {
+    const [isVisible, setIsVisible] = useState(true)
+    useEffect(() => {
```

Comment:
This component causes content shift. Also it doesn't really "fade-out"... I will add another task to utilize the [Radix Toast component](https://www.radix-ui.com/primitives/docs/components/toast) for any success or error notifications. It's fine to leave it here for now.

Reply:
In my demo I suggested putting it at the bottom where it doesn't cause content shift, and yes the 'fade' doen't work yet. I can implement the toast if you want.